modules/exploits/multi/ssh/sshexec.rb

```diff
@@ -35,39 +35,82 @@ def initialize
         },
       'Payload'          =>
         {
-          'Space'        => 4096,
+          'Space'        => 800000,
           'BadChars'     => "",
           'DisableNops'  => true
         },
-      'Platform'         => %w{ linux osx python },
+      'Platform'         => %w[linux osx python],
+      'CmdStagerFlavor'  => %w[bourne echo printf wget],
       'Targets'          =>
         [
-          [ 'Linux x86',
+          [
+            'Linux x86',
             {
               'Arch'     => ARCH_X86,
               'Platform' => 'linux'
             }
           ],
-          [ 'Linux x64',
+          [
+            'Linux x64',
             {
               'Arch'     => ARCH_X64,
               'Platform' => 'linux'
             }
           ],
-          [ 'OSX x86',
+          [
+            'Linux armle',
             {
-              'Arch'     => ARCH_X86,
-              'Platform' => 'osx'
+              'Arch'     => ARCH_ARMLE,
+              'Platform' => 'linux'
+            }
+          ],
+          [
+            'Linux mipsle',
+            {
+              'Arch'             => ARCH_MIPSLE,
+              'Platform'         => 'linux',
+              'CmdStagerFlavor'  => %w[curl wget]
+            }
+          ],
+          [
+            'Linux mipsbe',
+            {
+              'Arch'             => ARCH_MIPSBE,
+              'Platform'         => 'linux',
+              'CmdStagerFlavor'  => %w[wget]
+            }
+          ],
+          [
+            'Linux aarch64',
+            {
+              'Arch'     => ARCH_AARCH64,
+              'Platform' => 'linux'
             }
           ],
-          [ 'Python',
+          [
+            'OSX x86',
+            {
+              'Arch'             => ARCH_X86,
+              'Platform'         => 'osx',
+              'CmdStagerFlavor'  => %w[curl wget]
+            }
+          ],
+          [
+            'OSX x64',
+            {
+              'Arch'             => ARCH_X64,
+              'Platform'         => 'osx',
+              'CmdStagerFlavor'  => %w[curl wget]
```

Comment:
Should we limit this to `curl`?  Since `wget` is not available by default in macOS?

Reply:
Sure; it is all brushed metal dark magic to me.